config/rbac/auth_proxy_service.yaml

```diff
@@ -3,16 +3,16 @@ kind: Service
 metadata:
   annotations:
     prometheus.io/scrape: "true"
```

Comment:
Currently this service only works in combination with the `controller-manager-patch.yaml` from the observability folder.

I think it would be good to either:
* move the file to the observability folder
* make it functional be changing the bind addr in `config/manager/manager.yaml` (As mentioned: that's something which was rejected for security reasons in the main repo, but I'll stay out of this discussion :))

Reply:
@sbueringer thank you so much for your insights!

Reply:
You're welcome